Monika-After-Story/MonikaModDev
# Auto dark mode based on day cycle

~~Currently the auto mode of dark mode checks the current filter to determine which mode it should use. The issue is that, because the filter progression has to happen in the `spaceroom` label, auto mode doesn't work in the main menu and in the open door greeting.~~

~~This changes the auto mode and makes it check the current day cycle instead.~~

This adds a `mas_progressFilter` in the initial label of the open door greeting to update the filter and make the greeting use the correct styles when dark mode's auto mode is enabled.

## Testing:
- Verify that the proper styles are used in open door greeting when auto mode is enabled
- Verify that auto mode still works
- Verify no crashes
